mmv1/products/gkeonprem/VmwareCluster.yaml

```diff
@@ -0,0 +1,662 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: "VmwareCluster"
+min_version: beta
+base_url: "projects/{{project}}/locations/{{location}}/vmwareClusters"
+create_url: "projects/{{project}}/locations/{{location}}/vmwareClusters?vmware_cluster_id={{name}}"
+update_url: "projects/{{project}}/locations/{{location}}/vmwareClusters/{{name}}"
+delete_url: "projects/{{project}}/locations/{{location}}/vmwareClusters/{{name}}?force=true"
+self_link: "projects/{{project}}/locations/{{location}}/vmwareClusters/{{name}}"
+update_verb: :PATCH
+update_mask: true
+description: "A Google VMware User Cluster."
+autogen_async: false
```

Comment:
qq: We can't autogen this because of `validationCheck` right?

Reply:
Yes, we need to surface the validationCheck info in the operation response. Thanks for confirming